### PR TITLE
Update power grid example to be compatible with Julia 0.6.1

### PR DESCRIPTION
**Pull request by chldkdtn**
PowerGrid examples are now compatible with Julia 0.6.1. 

One thing to keep in mind to run power grid examples is that some lines in src/StructJuMP.jl have to be either commented or uncommented. It is documented in src/StructJuMP.jl file.

### CODE REVIEW

File: src/StructJuMP.jl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__precompile__()
+__precompile__()   # need to be commented for examples/PowerGrid to be run properly
```
Comment (joehuchette) [inline src/StructJuMP.jl]: Why is this the case?

File: src/StructJuMP.jl [hunk 2]
```diff
@@ -7,8 +7,8 @@ import MathProgBase
 import ReverseDiffSparse
 
 # These modules could be optional.
-# import StructJuMPSolverInterface
-# import MPI
+# import StructJuMPSolverInterface ## need to be uncommented for exmples/PowerGrid to be run properly
+# import MPI                       ## need to be uncommented for examples/PowerGrid to be run properly
```
Comment (joehuchette) [inline src/StructJuMP.jl]: Why is this the case?

Comment (cnpetra) [reply src/StructJuMP.jl]: by default, these imports are commented out in StructJuMP. I think the rationale is that when one uses StructJuMP in serial (without StructJuMPSolverInterface and PIPS), he/she does not have to clone StructJuMPSolverInterface and MPI. The imports are needed though for "parallel" StructJuMP models, such as those in examples/PowerGrid.

Comment (joehuchette) [reply src/StructJuMP.jl]: Ahh sorry, I missed that you were only adding comments to this effect. You shouldn't have to touch ``src`` files to make things work, but that can be addressed separately.

**Comment by joehuchette** [comment]
Thanks, @chldkdtn!

**Pull request merged**